content/en/docs/collector/getting-started.md

```diff
@@ -26,12 +26,14 @@ $ git clone git@github.com:open-telemetry/opentelemetry-collector-contrib.git; \
 Pull a docker image and run the collector in a container. Replace `{{% param collectorVersion %}}`
 with the version of the Collector you wish to run.
 
-{{< ot-tabs DockerHub ghcr.io >}}
+{{< ot-tabs DockerHub >}}
```

Comment:
the idea of #1823 was to have here the core & contrib release of the collector, so have still 2 tabs, but instead of DockerHub and ghcr.io they are "core" and "contrib" and the difference is that once you pull `otel/opentelmetry-collector` and once `otel/opentelemetry-collector-core`